### DISCUSSION TITLE
Face detection don't work because of SSL error

### DISCUSSION THREAD

**Discussion by emoriver** [immich-app/immich#8865] [Q&A] [answered]
I was not able to see faces pop-up under "explore" so I had a look in the machine_learning container log and saw a number of errors: I attach the file... any ideas appreciated
[_immich_machine_learning_logs (1).txt](https://github.com/immich-app/immich/files/15005559/_immich_machine_learning_logs.1.txt)

**Comment by jrasm91** [comment]
(MaxRetryError("HTTPSConnectionPool(host='huggingfa
                             ce.co', port=443): Max retries exceeded with url:  
                             /api/models/immich-app/buffalo_l/revision/main     
                             (Caused by SSLError(SSLCertVerificationError(1,    
                             '[SSL: CERTIFICATE_VERIFY_FAILED] certificate      
                             verify failed: self-signed certificate

huggingface.co is throwing an SSL issue. Probably fine bad configuration on your side.

**Reply by emoriver** [reply]
I didn't change anything about the configuration (except .env obviously)... which configuration do you mean?

I found an answer here but didn't change anything:
https://github.com/immich-app/immich/discussions/6417

**Reply by jrasm91** [reply]
Self signed certificate means the request is being intercepted and blocked by something local in your setup

**Reply by emoriver** [reply]
Could be... I'm trying the containerized set-up with docker... where to start doing debug??

**Reply by jrasm91** [reply]
Probably ping, curl, and nslookup the domain to see where the request is going to.

**Reply by emoriver** [reply, marked as answer]
Hi Jason, after MANY errors/retries I found that the problem was in the Turnkey Debian 11 image... believe it or not... I'm working on a proxmox host and somehow that image prevented the container to connect to the internet, switching to a standard Debian 12 image everything is working fine so far! Thanks
